### PR TITLE
Update global/emergency banner playbooks.

### PR DESCRIPTION
**Pull request by sengi**
- We no longer run Varnish.
- Update the troubleshooting steps to reflect that Jenkins is no longer involved, Static's template cache no longer persists across rollouts, etc.
- Simplify the structure to make the playbook steps easier to follow (e.g. separate non-essential / background information from playbook steps).
- Minor fixes for accuracy, clarity and house style.
- Add alt text descriptions for images.

Tested rendering locally with `SKIP_PROXY_PAGES=true NO_CONTRACTS=true bundle exec middleman server`.

### CODE REVIEW

File: source/manual/emergency-publishing.html.md [hunk 1]
```diff
@@ -7,172 +7,182 @@ section: Publishing
...
-The information for the emergency banner is stored in Redis. [Static](https://github.com/alphagov/static) is responsible for displaying the data and we run [rake task in static](https://github.com/alphagov/static/blob/main/lib/tasks/emergency_banner.rake) to set or delete the appropriate hash in Redis.
+- The information for the emergency banner is stored in a Redis key as a Ruby
+  hash (key-value map).
+- A [Rake task in
+  Static](https://github.com/alphagov/static/blob/main/lib/tasks/emergency_banner.rake)
+  sets or removes the Redis key.
+- Static is responsible for reading the data from Redis and rendering a partial
+  page from a template.
+- Frontend redering apps retrieve this partial page from Static via the Slimmer
+  library, which caches the response for up to 60 seconds.
```
Comment (theseanything) [inline source/manual/emergency-publishing.html.md:126]: Nice to have this behaviour written down!

File: source/manual/emergency-publishing.html.md [hunk 2]
```diff
@@ -7,172 +7,182 @@ section: Publishing
...
-- [https://www.gov.uk/search](https://www.gov.uk/search) ([Integration](https://www-origin.integration.publishing.service.gov.uk/search), [Staging](https://www-origin.staging.publishing.service.gov.uk/search))
+- [https://www.gov.uk/?ae00e491](https://www.gov.uk/?ae00e491) ([staging](https://www-origin.staging.publishing.service.gov.uk/?ae00e491))
+- [https://www.gov.uk/financial-help-disabled?7f7992eb](https://www.gov.uk/financial-help-disabled?7f7992eb) ([staging](https://www-origin.staging.publishing.service.gov.uk/financial-help-disabled?7f7992eb))
+- [https://www.gov.uk/government/organisations/hm-revenue-customs?49854527](https://www.gov.uk/government/organisations/hm-revenue-customs?49854527) ([staging](https://www-origin.staging.publishing.service.gov.uk/government/organisations/hm-revenue-customs?49854527))
+- [https://www.gov.uk/search?q=69b197b8](https://www.gov.uk/search?q=69b197b8) ([staging](https://www-origin.staging.publishing.service.gov.uk/search?q=69b197b8))
```
Comment (ChrisBAshton) [inline source/manual/emergency-publishing.html.md:102]: I would just omit the cachebust strings. If we're having to change them anyway, having them there isn't very helpful, and we already explain the need to add cachebust strings in the section above this one.

**Review by theseanything — APPROVED** [review]

**Review by ChrisBAshton — APPROVED** [review]
LGTM. Two nitpicks below 👍

File: source/manual/emergency-publishing.html.md (hunk 2 above)
Comment (sengi) [reply source/manual/emergency-publishing.html.md]: Yeah was thinking the same thing. Done :)

File: source/manual/emergency-publishing.html.md [hunk 3]
```diff
@@ -7,172 +7,182 @@ section: Publishing
...
 
-The information for the emergency banner is stored in Redis. [Static](https://github.com/alphagov/static) is responsible for displaying the data and we run [rake task in static](https://github.com/alphagov/static/blob/main/lib/tasks/emergency_banner.rake) to set or delete the appropriate hash in Redis.
+- The information for the emergency banner is stored in a Redis key as a Ruby
+  hash (key-value map).
+- A [Rake task in
+  Static](https://github.com/alphagov/static/blob/main/lib/tasks/emergency_banner.rake)
+  sets or removes the Redis key.
+- Static is responsible for reading the data from Redis and rendering a partial
+  page from a template.
+- Frontend redering apps retrieve this partial page from Static via the Slimmer
```
[the remark this answers is not in the document]
Comment (sengi) [reply source/manual/emergency-publishing.html.md]: Thanks! (I really must get around to fixing my editor spellcheck 😅)

**Pull request merged**